tests/st/calicoctl/test_crud.py

```diff
@@ -1479,369 +1479,366 @@ def test_get(self):
+                                'peerIP': '192.168.0.250',
+                                'scope': 'node'}
+                       # Valid numbers are <=4294967295
+                   }),
+                   ("bgpPeer-invalidIP", {
+                       'apiVersion': API_VERSION,
+                       'kind': 'BGPPeer',
+                       'metadata': {'name': 'bgppeer3'},
+                       'spec': {'asNumber': 64513,
+                                'node': 'Node1',
+                                'peerIP': '192.168.0.256',
+                                'scope': 'node'}
+                   }),
+                   ("bgpPeer-apiversion", {
+                       'apiVersion': 'v7',
+                       'kind': 'BGPPeer',
+                       'metadata': {'name': 'bgppeer4'},
+                       'spec': {'asNumber': 64513,
+                                'node': 'Node1',
+                                'peerIP': '192.168.0.250',
+                                'scope': 'node'}
+                   }),
+                   ("bgpPeer-invalidIpv6", {
+                       'apiVersion': API_VERSION,
+                       'kind': 'BGPPeer',
+                       'metadata': {'name': 'bgppeer5'},
+                       'spec': {'asNumber': 64590,
+                                'node': 'Node2',
+                                'peerIP': 'fd5f::6::ee',
+                                'scope': 'node'}
+                   }),
+                   ("bgpPeer-invalidNodename", {
+                       'apiVersion': API_VERSION,
+                       'kind': 'BGPPeer',
+                       'metadata': {'name': 'bgppeer6'},
+                       'spec': {'asNumber': 64590,
+                                'node': 'Node 2',
+                                'peerIP': 'fd5f::6:ee',
+                                'scope': 'node'}
+                   }),
+                   # See issue https://github.com/projectcalico/libcalico-go/issues/248
+                   ("bgpPeer-unrecognisedfield", {
+                       'apiVersion': API_VERSION,
+                       'kind': 'BGPPeer',
+                       'metadata': {'name': 'bgppeer7'},
+                       'spec': {'asNumber': 64590,
+                                'unknown': 'thing',
+                                'node': 'Node2',
+                                'peerIP': 'fd5f::6:ee',
+                                'scope': 'node'}
+                   }),
+                   # See issue https://github.com/projectcalico/libcalico-go/issues/222
+                   ("bgpPeer-longname", {
+                       'apiVersion': API_VERSION,
+                       'kind': 'BGPPeer',
+                       'metadata': {'name':'bgppeer8'},
+                       'spec': {'asNumber': 64590,
+                                'node':
+                                    'TestTestTestTestTestTestTestTestTestTestTest'
```

Comment:
This won't be testing the field length.

Since node is also a name field, we don't allow caps.